docs/release-notes/8.18.asciidoc

```diff
@@ -1,6 +1,25 @@
 [[release-notes-header-8.18.0]]
 == 8.18
 
+[discrete]
+[[release-notes-8.18.1]]
+=== 8.18.1
+
+[discrete]
+[[enhancements-8.18.1]]
+==== Enhancements
+* Updates the **MITRE ATT&CK® coverage** page mapping to `v16.1` ({kibana-pull}215026[#215026]).
+* Adds a background task to upgrade Agentless deployments after {kib} has been upgraded ({kibana-pull}207143[#207143]).
+
+[discrete]
+[[bug-fixes-8.18.1]]
+==== Fixes
+* Removes the check for unused connector roles ({kibana-pull}219358[#219358]).
+* Simplifies and improves the rule import error message ({kibana-pull}218701[#218701]).
+* Fixes the related integrations render performance on rule editing pages ({kibana-pull}217254[#217254]).
+* Adds process events for ptrace and memfd activity for Linux OSes running ebpf-based event sourcing (kernel versions 5.10 and newer).
```

Comment:
@nicholasberlin This didn't land in 8.18.1, right?

Reply:
> I also added and forgot to remove `v9.0.1`, is there a similar release note PR for that?

Not yet: https://github.com/elastic/security-docs/pulls?q=is%3Apr+is%3Aopen+label%3Arelease-notes

Reply:
```suggestion
```

Reply:
The 9.0.1 release notes are in a different repo. : ) 

https://github.com/elastic/docs-content/pull/1309

[Here's](https://github.com/elastic/docs-content/pull/1309/files#diff-08c97e20bb7e67e25fdec55430f346b95d5b1fde76faf034f7e44cc24129dfdbR28-R31) the list of bug fixes that made it into 9.0.1. As you can see, https://github.com/elastic/endpoint-dev/pull/15318 is not included.